a10_neutron_lbaas/a10_context.py

```diff
@@ -33,6 +33,7 @@ def __init__(self, handler, openstack_context, openstack_lbaas_obj,
         self.db_operations = self.a10_driver.db_operations_class(self.openstack_context)
         self.inventory = self.a10_driver.inventory_class(self)
         LOG.debug("A10Context obj=%s", openstack_lbaas_obj)
+        self.partition_name = None
```

Comment:
Yeah it gets set to shared further down the line.  We have various checks in place that check for a None value and I didn't want to interfere with those.